plugins/win-capture/graphics-hook/gl-capture.c

```diff
@@ -791,6 +794,9 @@ static BOOL WINAPI hook_swap_buffers(HDC hdc)
 
 static BOOL WINAPI hook_wgl_swap_buffers(HDC hdc)
 {
+	if (!gl_functions_initialized)
+		return FALSE;
```

Comment:
Style:  For consistency with other `return` statements, write "FALSE" in lowercase letters.